jenkins/fossa_install.sh

```diff
@@ -1,12 +1,35 @@
 #!/bin/bash
 
+BUILD_DIR="$WORKSPACE"
+CLI_INSTALL_SCRIPT="https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh"
+
+install_latest()
+{
+  echo "Installing Latest CLI Version..."
+  if ! curl -H 'Cache-Control: no-cache' $CLI_INSTALL_SCRIPT | bash -s -- -b "$BUILD_DIR"; then
```

Comment:
how about putting this cmd in a var so it can be reused in the install_pinned function?

Reply:
I'd have to play with exactly how to have the variable executed in an if statement rather than being treated as a string.  If you have an example I'd give it a shot, but these curl commands have been kind touchy... im reluctant to change them since they work.